impala/_thrift_api.py

```diff
@@ -103,6 +108,184 @@
     ThriftClient = TClient
 
 
+class ImpalaHttpClient(TTransportBase):
+  """Http implementation of TTransport base."""
```

Comment:
Maybe leave a comment with a pointer to indicate it's a copy of the code from impala shell?

Reply:
Thanks,I did this but I added a small change